test/test-run-express-metrics.js

```diff
@@ -0,0 +1,35 @@
+var helper = require('./helper');
```

Comment:
This test times out for me, I guess it passes for you? I wonder if there is a race condition between starting sl-run, and starting to listen for messages from it (I don't have time to check now, just a thought, if you are on a mac linux scheduling could be a difference).